Module.php

```diff
@@ -8,9 +8,9 @@ class Module
      */
     public function getAutoloaderConfig()
     {
-        return array('Zend\Loader\StandardAutoloader' => array('namespaces' => array(
+        return [\Zend\Loader\StandardAutoloader::class => ['namespaces' => [
             __NAMESPACE__ => __DIR__.'/src/',
```

Comment:
However: *do* import the StandardAutoloader class, instead of using global qualification.